components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -169,14 +192,14 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
     this.store.select(addPolicyMembersStatus).pipe(
       filter(identity),
       takeUntil(pendingAdd))
-      .subscribe((state) => {
-        if (state === EntityStatus.loadingSuccess) {
+      .subscribe((p_state) => {
```

Comment:
Small update to the variable name from `state` to `p_state` only because our new animation is importing a `state` variable and there is no variable shadowing allowed.

Reply:
Please use `pState` though to follow conventions (or `addPolicyState` might help, too).